src/test/java/uk/gov/hmcts/divorce/document/content/NoticeOfProceedingContentTest.java

```diff
@@ -264,17 +247,16 @@ public void shouldSuccessfullyApplyCivilPartnershipContentForNoticeOfProceedings
 
         var ctscContactDetails = CtscContactDetails
             .builder()
-            .centreName("HMCTS Digital Divorce and Dissolution")
-            .serviceCentre("Courts and Tribunals Service Centre")
             .poBox("PO Box 13226")
             .town("Harlow")
             .postcode("CM20 9UG")
-            .phoneNumber("0300 303 0642")
             .build();
 
         when(commonContent.getPartner(caseData, caseData.getApplicant2(), ENGLISH)).thenReturn(CIVIL_PARTNER);
         when(holdingPeriodService.getDueDateFor(LocalDate.of(2021, 6, 18)))
             .thenReturn(LocalDate.of(2021, 11, 6));
+        when(docmosisCommonContent.getBasicDocmosisTemplateContent(caseData.getApplicant1().getLanguagePreference()))
```

Comment:
Optional as I think it should have been done sooner but could you update the TestDataHelper.getBasicDocmosisTemplateContent method to include the ctsc vars?